plugin.json.md

```diff
@@ -0,0 +1,19 @@
+## plugin.json
```

Comment:
I thought this file could simply link to https://github.com/Flow-Launcher/Flow.Launcher.PluginsManifest.

Reply:
The manifest file is slightly different from the plugin.json, and the document kindly suggests that people already have the plugin.json.